api/stats_agg.md

```diff
@@ -0,0 +1,90 @@
+# stats_agg() <tag type="toolkit" content="toolkit" />
+An aggregate that produces a StatsSummary from `DOUBLE PRECISION` values. 
+
+Statistical aggregates can be done on either a single variable, in which case
+only one-dimensional aggregates are calculated or two variables, in which case
+one-dimensional aggregates are calculated for each and linear regression is performed
+on both together. 
+
+For more information about statistical aggregation functions, see the
+[hyperfunctions documentation][hyperfunctions-stats-agg].
+
+## 1D Statistical Aggregates
```

Comment:
```suggestion
## 1D Statistical Aggregates
When you perform a statistical aggregate on a single variable, a 
one-dimensional aggregate is calculated.
```

Just to avoid this being a naked heading.